mvcc/backend/backend.go

```diff
@@ -536,3 +557,16 @@ func (s *snapshot) Close() error {
 	<-s.donec
 	return s.Tx.Rollback()
 }
+
+func copyTxReadBuffer(from txReadBuffer) txReadBuffer {
```

Comment:
Declare this as `func (txr txReadBuffer) copy() txReadBuffer` instead?

Reply:
Good idea.